dispatcher/cli/dcos_spark/cli.py

```diff
@@ -8,6 +8,8 @@
     dcos spark run --help
     dcos spark run --submit-args=<spark-args>
                    [--dcos-space=<dcos_space>]
+                   [--kerberos-principal=kerberos_principal]
+                   [--keytab-secret-path=<keytab_secret>]
```

Comment:
@susanxhuynh I see, `--tgt` is a custom commit arg that is passed as a `spark-submit` arg. I updated `hdfs.md` with the new protocol.